### DISCUSSION TITLE
How to install/use corepack?

### DISCUSSION THREAD

**Discussion by rbirkgit** [yarnpkg/berry#5949] [General]
I cannot find any docs/instructions of how to start using corepack. 

I have been using yarn 3.x for some time and wanted to update to 4.x and try out corepack. I enabled it with `corepack enable` and then tried `yarn set version stable` and `yarn install` 

It downloaded 4.0.1 to my local folder and not using corepack. This is what yarn install outputs.

➤ YN0048: Automatically removed core plugins that are now builtins 👍
➤ YN0087: Migrated your project to the latest Yarn version 🚀

I cannot for the life of me understand how to enable and start using corepack. I think the yarn docs could be improved in this area.

**Comment by forcemagic** [comment]
Hi!

I believe that the documentation at [the official repo](https://github.com/nodejs/corepack) and [at Node.js](https://nodejs.org/api/corepack.html) is quite good, quality-wise.

By running `yarn set version stable`, yarn downloads `yarn@stable` and puts it in your `.yarn` folder. In order to use a global and *modern* yarn, try running `corepack use yarn@stable` as described in the docs, but make sure to remove the `yarnPath` directive from `.yarnrc.yml` as that will still allow yarn to resolve to a the workspace version.

For me, deleting `.yarn/releases/*` and removing `yarnPath` from `yarnrc` worked just fine with the command above, and now corepack uses yarn 4.0.2. Make sure you have `packageManager` properly set in your `package.json`! (Corepack does this for you afaik)

Hope this helps!
